huggingface/tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -17,7 +17,10 @@ RUN pip install --no-cache-dir \
     transformers[sklearn,sentencepiece]==${TRANSFORMERS_VERSION} \
     datasets==${DATASETS_VERSION} \
     tensorflow-addons==0.12.0 \
-    psutil
+    psutil \
+	 # TODO: Remove upgrade statements once packages are updated in base image
+	"Pillow>=8.3.2"
```

Comment:
nit: I think this indentation is a bit off

Reply:
done